tests/console/glibc_tunables.pm

```diff
@@ -0,0 +1,58 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 Guillaume GARDET
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Package: glibc
+# Summary: Check GLIBC_TUNABLES support
+# Maintainer: Guillaume GARDET <guillaume@opensuse.org>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils 'zypper_call';
+
+sub run {
+    select_console 'root-console';
+
+    if (check_var('ARCH', 'aarch64') && check_var('AARCH64_MTE_SUPPORTED', '1')) {
```

Comment:
Please add `AARCH64_MTE_SUPPORTED`  to [variables.md]( https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/variables.md)

and use is_aarch64 (`use Utils::Architectures 'is_aarch64'`)

Reply:
Done